provisioner/ansible/provisioner.go

```diff
@@ -336,6 +336,13 @@ func (p *Provisioner) executeAnsible(ui packer.Ui, comm packer.Communicator, pri
 		// args = append(args, "--private-key", privKeyFile)
 		args = append(args, "-e", fmt.Sprintf("ansible_ssh_private_key_file=%s", privKeyFile))
 	}
+
+	// expose packer_http_addr extra variable
+	httpAddr := common.GetHTTPAddr()
+	if httpAddr != "" {
+		args = append(args, "-e", fmt.Sprintf("packer_http_addr=%s", httpAddr))
```

Comment:
Always use long arga.

Reply:
I'm sorry i didn't understand what you meant.
I guess it's a typo so `always use long args`, but even with that, I'm totally new to Packer source code and Go.

I basically copy pasted these 3 lines from another PR...

Reply:
did you mean to use `--extra-vars` ? (long command line args)
I guess so.
but this line above was using the short version:
`args = append(args, "-e", fmt.Sprintf("ansible_ssh_private_key_file=%s", privKeyFile))`